feature-libs/order/README.md

```diff
@@ -2,4 +2,8 @@
 
 Spartacus Order feature can be added to the existing Spartacus application by running `ng add @spartacus/order`. For more information about Spartacus schematics, visit the [official Spartacus schematics documentation page](https://sap.github.io/spartacus-docs/schematics/).
 
+To see enhanced UI in Order History and Order Details page, 
+1. Add `{ provide: MYACCOUNT_ORDER_ENHANCED_UI, useValue: true },` to order-feature.module.ts file `providers` array.
+2. Add `pdf-invoices` feature to the storefront, if not installed already.
```

Comment:
[MAJOR]
Does it mean, that for the New-MyAccount feature, the "PDF invoices" feature library will becomes a mandatory library to be installed? 
If yes, then also our installation documentation for New-MyAccount (and it's schematics) should reflect that.

Reply:
Yes, PDF Invoices will become a mandatory library. But if its not installed, it will not break the UI. When the user clicks on 'Download Invoices' button in Order Details screen, it will show spinner and no data will load there. If user wants to see data inside that dialog, PDF Invoices should be installed. Since this new UI is controlled based on token when user can set only after installing the storefront, i have not altered the schematics to download PDF Invoices as well. I thought we can document that- if user wants to use the new UI for orders, then add PDF Invoices feature to your storefront.

Reply:
Yes. let's document it. 
If/When new MyAccount will become the default one, our default installation schematics might need to install PDF library. Docs will help to remind us in the feature that we need to do it. At least there will be a written track about it.